hal/src/msom/hal_platform_config.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "hal_platform_rtl8721x_config.h"
+#include "platforms.h"
+
+#define HAL_PLATFORM_NCP (1)
+#define HAL_PLATFORM_NCP_AT (1)
+#define HAL_PLATFORM_CELLULAR (1)
+#define HAL_PLATFORM_CELLULAR_SERIAL (HAL_USART_SERIAL3)
+#define HAL_PLATFORM_SETUP_BUTTON_UX (1)
+#define HAL_PLATFORM_SPI_NUM (2)
+#define HAL_PLATFORM_I2C_NUM (1)
+#define HAL_PLATFORM_I2C_BUFFER_SIZE(x) (512)
+#define HAL_PLATFORM_USART_NUM (3)
+#define HAL_PLATFORM_NCP_COUNT (2)
+#define HAL_PLATFORM_WIFI (1)
+#define HAL_PLATFORM_WIFI_COMPAT (1)
+#define HAL_PLATFORM_WIFI_SCAN_ONLY (0)
+
+#define HAL_PLATFORM_PMIC_BQ24195 (1)
+#define HAL_PLATFORM_PMIC_BQ24195_I2C (HAL_I2C_INTERFACE1)
+#define HAL_PLATFORM_FUELGAUGE_MAX17043 (1)
+#define HAL_PLATFORM_FUELGAUGE_MAX17043_I2C (HAL_I2C_INTERFACE1)
+
+#define HAL_PLATFORM_POWER_MANAGEMENT (1)
+#define HAL_PLATFORM_POWER_MANAGEMENT_OPTIONAL (1)
+
+#define HAL_PLATFORM_RADIO_ANTENNA_INTERNAL (0)
+#define HAL_PLATFORM_RADIO_ANTENNA_EXTERNAL (0)
```

Comment:
Does this need to be set to '1'?

Reply:
(I posted this comment previously but it got deleted somehow)

Its a good question. I think `HAL_PLATFORM_RADIO_ANTENNA_EXTERNAL` is intended for platforms that have **both** an internal **and** external wifi / ble antenna ( ie P2, Argon, etc). This define allows ``selectAntenna()`` to control the IO needed to switch between an external and internal antenna for BLE/WIFI.
MSoM does not appear to have a switch, so in effect it only has the external BLE/WIFI antenna. So it just always defaults to that and calling
`selectAntenna()`
will return
`SYSTEM_ERROR_NOT_SUPPORTED`
So I dont think we need to enable these options.

Reply:
As long as `SYSTEM_ERROR_NOT_SUPPORTED` is returned and `getAntenna()` returns the external one then we can pass over this setting.